kv/kv.go

```diff
@@ -494,6 +494,10 @@ type Storage interface {
 	Describe() string
 	// ShowStatus returns the specified status of the storage
 	ShowStatus(ctx context.Context, key string) (interface{}, error)
+}
+
+// Store defines the interface for a Store.
+type Store interface {
```

Comment:
I think it is somehow difficult to distinguish the difference between `Storage` and `Store`. How about renaming it to `StorageEx` (which means extended `Storage`). And we can embed `Storage` interface too.

Reply:
like this:
```
type StorageEx interface {
    Storage
    GetMemCache() MemManager
}
```